src/pystatis/config.py

```diff
@@ -33,6 +33,20 @@
     "zensus": re.compile(r"^\d{4}[A-Z]-\d{4}$"),
     "regio": re.compile(r"^((\d{5}-.{1,2}($|-.*$))|(A.*$)|([0-9A-Z]{10}$)|(\d{5}\w-Z-\d{1,2}))"),
 }
+ZENSUS_AGS_CODES = [
+    "GEOBL1",
+    "GEOBL3",
+    "GEOGM1",
+    "GEOGM2",
+    "GEOGM3",
+    "GEOLK1",
+    "GEOLK3",
+    "GEORB1",
+    "GEORB3",
+    "GEOVB1",
+    "GEOVB3",
+]
+REGIO_AGS_CODES = ["DLAND", "DLANDU", "REGBEZ", "KREISE", "GEMEIN"]
```

Comment:
If these codes are the same for regio and "general" genesis, the name is slightly misleading (too narrow in terms of scope) at first glance. How about REGIO_(AND_)GENESIS_AGS_CODES? Or adding a comment clarifying this?

Reply:
Not really as these codes CAN appear in Genesis, but Genesis has no use of `regionalvariable` and `regionalkey`. That is why I called it REGIO_AGS_CODES, because in Genesis you always have different Tables and thus EVAS numbers, but in Regio you can select the regional variable yourself for tables ending with "B"

Reply:
But I guess we could call it Genesis AGS Codes, but still Zensus is different but it's also a Genesis instance...

Reply:
Done